### PR TITLE
Add some QingStor manual pages

### PR DESCRIPTION
**Pull request by markduan**

### CODE REVIEW

File: content/storage/object-storage/manual/bucket_manage.md [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+
+对象存储的主视图是 Bucket 列表，展示当前区域下用户所有的 Bucket 信息，包括名称、当前文件数、空间大小、权限以及可访问的 url。
+
+- 名称：该 Bucket 在青云对象存储服务中全局唯一的标记。
+- 当前文件数：Bucket 中所有 Object 的数量，包括文件夹在内。
+- 空间大小：Bucket 中所有 Object 大小的总和，是 Bucket 的一个监控项和计费项。
+- 权限：该 Bucket 的可访问性，控制台中可以设为私有、公开只读、公开读写。
+- url：Bucket 通过 RESTful API 访问时的 uri，调用 Object 相关 API 时的 uri 前缀。
+
+[![](_images/bucket_listview.png)](_images/bucket_listview.png)
```
Comment (Xuanwo) [inline content/storage/object-storage/manual/bucket_manage.md:22]: I prefer put images into `static/storage/object-storage/manual`, so we can use

```
![](bucket_listview.png)
```

File: content/storage/object-storage/news/announcements.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: "公告"
+date: 2020-01-30T00:38:25+09:00
+description: todo Test description
+draft: false
+enableToc: false
+---
+
+## lorem dollor sit amet
```
Comment (Xuanwo) [inline content/storage/object-storage/news/announcements.md:9]: remove this file

**Review by Xuanwo — COMMENTED** [review]
rest LGTM

File: content/storage/object-storage/news/announcements.md (hunk 2 above)
Comment (markduan) [reply content/storage/object-storage/news/announcements.md:9]: 这个需要补充的，暂时留在这里占位

**Review by Xuanwo — APPROVED** [review]

**Pull request merged**